packages/components/tests/dummy/app/templates/components/form/base-elements.hbs

```diff
@@ -905,19 +1013,31 @@
   <span class="dummy-text-small">With simple text</span>
   <Hds::Form::Label>This is a simple label</Hds::Form::Label>
   <br />
+  <span class="dummy-text-small">With required badge</span>
+  <Hds::Form::Label @isRequired={{true}}>This is the label</Hds::Form::Label>
+  <br />
+  <span class="dummy-text-small">With optional hint</span>
+  <Hds::Form::Label @isOptional={{true}}>This is the label</Hds::Form::Label>
+  <br />
   <span class="dummy-text-small">With structured content (eg. a
     <code>flex</code>
     layout and a
     <code>&lt;Badge&gt;</code>)</span>
   <Hds::Form::Label>
     <div class="dummy-form-base-elements-label-with-badge">This is the label
-      <Hds::Badge @size="small" @text="Some badge" /></div>
+      <Hds::Badge @size="small" @text="Some badge" />
```

Comment:
[question/suggestion] do you think we should showcase an example of "With structured content + Required" and/or "With structured content + Optional" (probably with short text and with long text)? On one side would be nice so we can test it works as expected, but on the other I fear it would confuse the end consumers...

Reply:
Seeing Heather backed this suggestion I added a few more examples.